### ISSUE TITLE
GUI freezes

### ISSUE THREAD

**Issue by neurochen** [open-ephys/plugin-GUI#537]
A bug was encountered while recording. The GUI freezed without any message box showing up on WIndows 10. We are recording 192 channels, and the SSD disk still has more than 1TB space. It freeezed in the middle of recording (see snapshot below). Any clues? The GUI stop showing any neural signal visualizations, all of them became static. But we could still remotely start/stop recordings; in this case, it created new files for the record nodes, but no data is logged into these files.
![Untitled](https://user-images.githubusercontent.com/10821596/203797726-c442d37d-cd33-439b-9a1c-11467e987500.png)

Update: based on the monitor, the hardware is till working. We still hear spikes in speaker, but the GUI and recording stopped.

**Comment by neurochen** [comment]
>

**Comment by neurochen** [comment]
Update:
![image](https://user-images.githubusercontent.com/10821596/204370395-38414ad4-4516-4db4-8c5f-646ed32fa171.png)

**Comment by jsiegle** [comment]
This looks like a USB streaming error. Are there any other applications running on the same computer during data acquisition?

**Comment by neurochen** [comment]
Yes, there is a teensy 3.5 and a FLIR USB camera. What are the recommended settings fro running plugin-GUI under windows 10? If I drive the board using Intan RHX, it seemed fine for long periods of recordings. But Intan RHX can't get the analog inputs.

**Comment by jsiegle** [comment]
There are some subtle differences between the Intan API and the one used by Open Ephys, but I'm not sure if they could explain these errors.

Do you ever experience freezes if the Teensy and USB camera are not acquiring data? In general, I would recommend acquiring these data streams on a separate computer if at all possible.

**Comment by jsiegle** [comment]
If this problem still persists with GUI v1.0, please open a new issue.
